tests/src/TestShapeMocks.cpp

```diff
@@ -1,9 +1,10 @@
 #define CATCH_CONFIG_MAIN // This tells Catch to provide a main() - only do this
                           // in one cpp file
+#include "Shape_mocks.h"
+
 #include <catch.hpp>
```

Comment:
Prefer to include standard library headers after local headers as it ensures headers compile on their own.